inputs/elements/working_example2.json

```diff
@@ -0,0 +1,615 @@
+                    "value": "0"
+                },
+                "crate": {
+                    "unit": "factor of total capacity (kWh)",
+                    "value": "1"
+                },
+                "efficiency": {
+                    "unit": "factor",
+                    "value": "0.9500000000000001"
+                },
+                "installedCap": {
+                    "unit": "kWh",
+                    "value": "0"
+                },
+                "label": "discharging_power",
+                "lifetime": {
+                    "unit": "year",
+                    "value": "10"
+                },
+                "opex_fix": {
+                    "unit": "currency/kWh/year",
+                    "value": "0"
+                },
+                "opex_var": {
+                    "unit": "currency/kWh",
+                    "value": "0"
+                },
+                "soc_initial": {
+                    "unit": "None or factor",
+                    "value": NaN
+                },
+                "soc_max": {
+                    "unit": "factor",
+                    "value": NaN
+                },
+                "soc_min": {
+                    "unit": "factor",
+                    "value": NaN
+                },
+                "unit": "kW"
+            },
+            "inflow_direction": "ESS Li-Ion",
+            "label": "ESS Li-Ion",
+            "optimizeCap": "True",
+            "outflow_direction": "ESS Li-Ion",
+            "storage_filename": "storage01.csv",
+            "type_oemof": "storage"
+        }
+    },
+    "engineering": {
+        "capex_fix": {
+            "unit": "currency",
+            "value": "0"
+        },
+        "capex_var": {
+            "unit": "currency",
+            "value": "0"
+        },
+        "label": "R&D, engineering",
+        "lifetime": {
+            "unit": "year",
+            "value": "20"
+        },
+        "opex_fix": {
+            "unit": "currency/year",
+            "value": "0"
+        },
+        "opex_var": {
+            "unit": "currency/kWh",
+            "value": "0"
+        }
+    },
+    "fixcost": {
+        "distribution_grid": {
+            "age_installed": {
+                "unit": "year",
+                "value": "10"
+            },
+            "capex_fix": {
+                "unit": "currency",
+                "value": "0.0"
+            },
+            "capex_var": {
+                "unit": "currency",
+                "value": "0.0"
+            },
+            "label": "distribution grid infrastructure",
+            "lifetime": {
+                "unit": "year",
+                "value": "30"
+            },
+            "opex_fix": {
+                "unit": "currency/year",
+                "value": "0"
+            },
+            "opex_var": {
+                "unit": "currency/kWh",
+                "value": "0"
+            }
+        }
+    },
+    "operation": {
```

Comment:
In working_example2, the "fixCosts" are not displayed correctly:
"engineering" and ""operation" are on the first level of the json, not below "fixCosts" on the second level.